docs/RUNNING-LOCALLY.md

```diff
@@ -20,7 +20,7 @@ Prerequisites
 - [Rimraf](https://github.com/isaacs/rimraf)
 - [Asar](https://github.com/electron/asar)
 - [NSIS v2.51](http://nsis.sourceforge.net/Main_Page) (v3.x won't work)
-- [Visual Studio Community 2013](https://www.visualstudio.com/en-us/news/vs2013-community-vs.aspx)
+- [Visual Studio Community 2015](https://go.microsoft.com/fwlink/?LinkId=691978&clcid=0x409) (free) (other editions, like Professional and Enterprise, should work too)
```

Comment:
Annoyingly, all the links on visualstudio.com seem to be 'generic', which as @WasabiFan points out means the versions they refer to could change.

But after a bit of digging around, I found this link, which I think could be what we're looking for?
https://www.microsoft.com/en-us/download/details.aspx?id=48146